ui/src/app/tasks/tasks-utils/tasks-import/tasks-import.component.ts

```diff
@@ -42,7 +42,7 @@ export class TasksImportComponent {
    * Import options
    */
   options = {
-    optimize: false
+    excludeChildren: true
```

Comment:
Wrong way around. Now you have a checkbox enabled to exclude but childs are not excluded. Not sure why we even want this?